### PR TITLE
Registration_Functionality

### PR DESCRIPTION
**Pull request by Vijay1090**
Hii VijayDYaligar, Please review Registration Functionality object repository, and Test Class for the same.

### CODE REVIEW

File: src/main/java/ObjectRepositorys/RegistretionPage.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.Select;
+import org.testng.Assert;
+
+public class RegistretionPage {
+	public void TC() {
+		System.out.println("I have added Registetion page");
+	}
```
Comment (VijayDYaligar) [inline src/main/java/ObjectRepositorys/RegistretionPage.java:10]: you have to add some remaining remaining xpath of element with its method

File: src/test/java/com/tests/RegistrationTest.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+public class RegistrationTest {
+	public void testCase1() {
+		System.out.println("Verified Registration logo");
+	}
+   
+	public void testCase2() {
+		System.out.println("Verified all registration element");
+	}
+	public void testCase3() {
```
Comment (VijayDYaligar) [inline src/test/java/com/tests/RegistrationTest.java:11]: these test cases seems good. no need to update.

**Review by VijayDYaligar — COMMENTED** [review]
I have viewed your working files. you have to need to do some refactory in Registration object repository file. and also do changes in test class as per object repository changes.

File: src/main/java/ObjectRepositorys/RegistretionPage.java (hunk 1 above)
Comment (Vijay1090) [reply src/main/java/ObjectRepositorys/RegistretionPage.java:10]: Hello VIjayDYaligar I have done changes as per your suggetion please review once.

**Review by VijayDYaligar — COMMENTED** [review]
Vijay1090 now it looks good. we can merge this in master.

**Pull request merged**